include/libm2k/contextbuilder.hpp

```diff
@@ -107,6 +119,7 @@ LIBM2K_API Context* deviceOpen()
 	return ContextBuilder::deviceOpen();
 }
 
+
```

Comment:
nitpick: extra line

Reply:
The line was added in order to be style-consistent.